### PR TITLE
use sdn workflow for aws-sdn-upgrade job

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml [hunk 1]
```diff
@@ -446,7 +446,7 @@ tests:
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
       TEST_TYPE: upgrade-conformance
-    workflow: openshift-upgrade-aws
+    workflow: openshift-upgrade-aws-sdn
```
Comment (bparees) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:449]: was this only wrong in 4.12 (not in 4.11?)

Comment (bparees) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:449]: (looks like there was no 4.11 equivalent to this job)

Comment (jluhrsen) [reply ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml:449]: and, 4.11 would use sdn by default as well.

**Pull request merged**